pytest_jira.py

```diff
@@ -258,8 +284,9 @@ def pytest_addoption(parser):
     group.addoption('--jira-no-ssl-verify',
                     action='store_false',
                     dest='jira_verify',
-                    default=verify,
-                    help='Disable SSL verification to Jira'
+                    default=_get_bool(config, 'DEFAULT', 'ssl_verification',
+                                      True),
```

Comment:
please use just one intend block, you can also move all that params on new line, like:

```
...
    default=_get_bool(
        config, 'DEFAULT', 'ssl_verification', True,
    )
...
```